lib/cosmos/dart/script/dart_ingester.rb

```diff
@@ -0,0 +1,52 @@
+require File.expand_path('../../config/environment', __FILE__)
+require 'dart_packet_log_writer'
+require 'dart_logging'
+
+class DartInterfaceThread < Cosmos::InterfaceThread
+  attr_writer :packet_log_writer
+  attr_writer :log_type
+
+  def handle_packet(packet)
+    if packet.target_name == 'SYSTEM'.freeze and packet.packet_name == 'META'.freeze
+      Cosmos::Logger.info("#{@log_type}: #{packet.target_name} #{packet.packet_name}")
+      # Update Current Value Table Used By Packet Log Writer
+      cvt_packet = Cosmos::System.telemetry.update!(packet.target_name, packet.packet_name, packet.buffer)
+      cvt_packet.received_time = packet.received_time
+      @packet_log_writer.start
```

Comment:
So when a new SYSTEM META packet arrives we create a new log file. Is this just to prevent large files or do other parts of DART rely on this? For example the DartPacketLogWriter?

Reply:
It's just to keep all the data in a file to the same META configuration.  It is not technically required as the readers and stream receivers should change configuration if the META configuration changes mid stream.